axolotlsay-js/dist.js

```diff
@@ -0,0 +1,30 @@
+const { bin } = require("./package.json");
+const execSync = require('child_process').execSync;
+
+// Compute the target we're building for
+const bunTargets = {
+    "x86_64-pc-windows-msvc": "bun-windows-x64",
+    "aarch64-apple-darwin": "bun-darwin-arm64",
+    "x86_64-apple-darwin": "bun-darwin-x64",
+    "aarch64-unknown-linux-gnu": "bun-linux-arm64",
+    "x86_64-unknown-linux-gnu": "bun-linux-x64"
```

Comment:
Double checked, and this list of targets hasn't changed since the bash was written - still no arm64 Windows, still no musl.